src/lib/Components/ArtworkGrids/RelayConnections/GeneArtworksGrid.tsx

```diff
@@ -16,15 +16,15 @@ const GeneArtworksGrid = createPaginationContainer(
           cursor: { type: "String" }
           sort: { type: "String", defaultValue: "-partner_updated_at" }
           medium: { type: "String", defaultValue: "*" }
-          priceRange: { type: "String", defaultValue: "*-*" }
+          price_range: { type: "String", defaultValue: "*-*" }
```

Comment:
I opted to move _all_ query vars to just use `price_range` as this is what metaphysics uses at the end (and what Eigen sends into Emission, and what force uses for hrefs)